packages/squiggle-lang/src/library/registry/frTypes.ts

```diff
@@ -236,6 +225,14 @@ export function frOr<T1, T2>(
   };
 }
 
+//TODO: It would probably eventually be good to refactor this out, to use frOr instead. However, that would be slightly less efficient.
```

Comment:
It'll be interesting to compare the performance (later). It's possible that checking for `v.tag === 1` is faster than `v instanceof BaseDist`.